yt_dlp/extractor/abematv.py

```diff
@@ -248,33 +187,33 @@ def mix_twist(nonce):
         return base64.urlsafe_b64encode(tmp).rstrip(b'=').decode('utf-8')
 
     def _get_device_token(self):
-        if self._USERTOKEN:
-            return self._USERTOKEN
+        if AbemaTVBaseIE._USERTOKEN:
+            return AbemaTVBaseIE._USERTOKEN
 
-        self._DEVICE_ID = str(uuid.uuid4())
-        aks = self._generate_aks(self._DEVICE_ID)
+        AbemaTVBaseIE._DEVICE_ID = str(uuid.uuid4())
+        aks = self._generate_aks(AbemaTVBaseIE._DEVICE_ID)
         user_data = self._download_json(
             'https://api.abema.io/v1/users', None, note='Authorizing',
             data=json.dumps({
-                'deviceId': self._DEVICE_ID,
+                'deviceId': AbemaTVBaseIE._DEVICE_ID,
                 'applicationKeySecret': aks,
             }).encode('utf-8'),
             headers={
                 'Content-Type': 'application/json',
             })
-        self._USERTOKEN = user_data['token']
+        AbemaTVBaseIE._USERTOKEN = user_data['token']
 
         # don't allow adding it 2 times or more, though it's guarded
         remove_opener(self._downloader, AbemaLicenseHandler)
         add_opener(self._downloader, AbemaLicenseHandler(self))
 
-        return self._USERTOKEN
+        return AbemaTVBaseIE._USERTOKEN
 
     def _get_media_token(self, invalidate=False, to_show=True):
-        if not invalidate and self._MEDIATOKEN:
-            return self._MEDIATOKEN
+        if not invalidate and AbemaTVBaseIE._MEDIATOKEN:
+            return AbemaTVBaseIE._MEDIATOKEN
 
-        self._MEDIATOKEN = self._download_json(
+        AbemaTVBaseIE._MEDIATOKEN = self._download_json(
```

Comment:
I prefer to use the class name only when necessary. Like:
```py
        if not invalidate and self._MEDIATOKEN:
            return self._MEDIATOKEN

        AbemaTVBaseIE._MEDIATOKEN = self._download_json(
```
But if you prefer to use `AbemaTVBaseIE` everywhere, that's fine

Reply:
> I prefer to use the class name only when necessary. 

Let's go with it